app/models/move.rb

```diff
@@ -137,6 +139,9 @@ def rebook
       date_from: date_from && date_from + 7.days,
       date_to: date_to && date_to + 7.days,
     )
+    new_move.determine_supplier
```

Comment:
You can remove this line now - isn't needed and the method will be removed. See https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/905